tokenserver/assignment/sqlnode/sql.py

```diff
@@ -608,7 +613,7 @@ def get_best_node(self, service):
                              nodes.c.capacity > nodes.c.current_load,
                              nodes.c.downed == 0)
                 fields = {
-                    'available': sqlfunc.min(
+                    'available': self._sqlfunc_min(
```

Comment:
Surprising no-one, running the MySQL tests in CI appears to have uncovered a bug in the MySQL support.  I don't think the previous version of this can ever have worked...perhaps we just don't exercise this code path very often in production?